src/reference/asciidoc/kafka.adoc

```diff
@@ -164,6 +164,26 @@ public KafkaTemplate<Integer, String> kafkaTemplate() {
 ----
 ====
 
+or if you are using Spring Boot, you can make use of the `KafkaProperties` ConfigurationProperties class to pull in application properties, rather than manually adding them to a map:
```

Comment:
This doesn't say anything  valuable to me.
Why would we mention Spring Boot in this case?

Reply:
True, and happy to remove it. I was just struggling to find a good Spring reference for using spring-kafka with Spring Boot. I see lots of teams in my organisation either writing loads of code manually copying properties wired in from application.yml into a map even though they could just use KafkaProperties, or else going out and copy/pasting any old code they find on StackOverflow. Reading through this documentation I can see how it would easily lead to the former approach.